exp-fashion-mnist/fashionmnist-train.py

```diff
@@ -103,51 +94,21 @@ def _prep():
     client.run(_prep)
 
     def submit(seed, **kwargs):
-        # import train
-
-        # assert train.__version__ == "0.1"
-
-        # import adadamp
-        #
-        # assert adadamp.__version__ == "0.2.0rc4"
-
         return train.main(
             epochs=epochs,
             verbose=True,
-            init_seed=seed,
-            random_state=seed,
-            tuning=True,
+            tuning=False,
```

Comment:
I think we should keep the seeds:

```suggestion
            init_seed=seed,
            random_state=seed,
            tuning=False,
```